.github/workflows/master.yml

```diff
@@ -179,7 +179,6 @@ jobs:
         run: >-
           ./build/mvn clean install -Dmaven.javadoc.skip=true -Drat.skip=true -Dscalastyle.skip=true -Dspotless.check.skip -V
           -pl kubernetes/integration-tests -am
```

Comment:
`-pl kubernetes/integration-tests -am` -> `-pl integration-tests/kyuubi-kubernetes-it -am`